.husky/pre-commit

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+# generate graphql schema
```

Comment:
Would suggest pushing this to CI.
While it's annoying when CI catches you, this sort of thing will lead to `--no-verify` being used. 

Or this may be better as a pre-push hook ? 
Some people commit quite often.